actors/evm/src/interpreter/address.rs

```diff
@@ -0,0 +1,88 @@
+use crate::interpreter::{H160, H256};
+use crate::U256;
+use fvm_shared::address::Address as FilecoinAddress;
+
+/// A Filecoin ad   dress as represented in the FEVM runtime.
+/// This type is a little bit silly now, but it will eventually handle f4 address detection.
+pub struct Address(H160);
```

Comment:
We also need a `From<FilecoinAddress>` impl i think.

Reply:
Yep, now is not a bad time to implement it and use it for the caller, receiver and the originator syscalls!

Reply:
Done in 1cbcd64.